ReSharper.FSharp/test/data/parsing/Type member - Member - XmlDoc 02.fs

```diff
@@ -0,0 +1,5 @@
+type A =
+    /// B
+    member x.B 
+        with get() = 5
+        and set (_: int) = ()
```

Comment:
Do we need the accessors in this XmlDoc test?

Reply:
Yes, since the problem was in properties with accessors
https://github.com/dotnet/fsharp/pull/12783

Reply:
So it requires two accessors with and a type annotation?

Reply:
I think one getter will be enough